src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -327,9 +327,6 @@ function New-AzConnectedKubernetes {
         # If GatewayResourceId is provided then set the gateway as enabled.
         $PSBoundParameters.Add('GatewayEnabled', -not [String]::IsNullOrEmpty($GatewayResourceId))
 
-        # !!PDS: Why are we only adding specific parameter here?  Isn't it better
```

Comment:
It looks like these parameters are those required in [README](https://github.com/xw-zhang24/azure-powershell/blob/main/src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/README.md#usage), emphasize I think.